### DISCUSSION TITLE
rkj theme with 24h clock.

### DISCUSSION THREAD

**Discussion by spitfire** [ohmyzsh/ohmyzsh#9822] [Help] [answered]
Im using raj theme which really suits me, but having 12h times really annoys me. How can I change the format to display 24h time?

**Comment by mcornella** [comment, marked as answer]
You can customize the theme by [copying it to the `$ZSH_CUSTOM`](https://github.com/ohmyzsh/ohmyzsh/wiki/Customization#overriding-and-adding-themes). Then change the `%I` in `%D{"%Y-%m-%d %I:%M:%S"}` for an `%H`.

**Reply by mcornella** [reply]
From https://github.com/ohmyzsh/ohmyzsh/discussions/9822#discussioncomment-661875:
> Is there a way to override it (In my .zshrc file or something else?) so I can still be able to get all updates?

You can programatically replace the `%I` in your zshrc file like so:
```zsh
PROMPT="${PROMPT:s/%I/%H}"
```

Put this at the end of your zshrc file, after OMZ is sourced.

EDIT: this also works:
```zsh
PROMPT="${PROMPT/\%I/%H}"
```

**Reply by spitfire** [reply]
Thanks a lot, that worked out great!
I still don't understand reasons using 12h format on computers though ¯\_(ツ)_/¯

**Comment by spitfire** [comment]
Is there a way to override it (In my .zshrc file or something else?) so I can still be able to get all updates?
